Tests/Svg.UnitTests/Svg.UnitTests.csproj

```diff
@@ -11,37 +11,6 @@
     <NoWarn>1591</NoWarn>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;NETFULL;NET452</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net452|AnyCPU'">
-    <DefineConstants>TRACE;RELEASE;NETFULL;NET452</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net462|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;NETFULL;NET462</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net462|AnyCPU'">
-    <DefineConstants>TRACE;RELEASE;NETFULL;NET462</DefineConstants>

```

Comment:
.NET 4.6.2 is supported until 2027, so I would leave these in.

Reply:
.NET Framework 4.6.2 is still valid.
https://github.com/svg-net/SVG/blob/71ffa1148a29eb1aab29ec09a09a98de2b5b7ddd/Tests/Svg.UnitTests/Svg.UnitTests.csproj#L4

I remove definition of symbols defined in vs project, or unused.

Reply:
True, you are right.